README.md

```diff
@@ -54,6 +54,13 @@ LoggingSystem.bootstrap(StreamLogHandler.standardError)
 
 For further information, please check the [API documentation][api-docs].
 
+#### Selecting a logging backend implementation (applications only)
+
+As the API has just launched, not many implementations exist yet. If you are interested in implementing one see the "Implementation considerations" section below explaining how to do so. List of existing SwiftLog API compatible libraries:
+
+- [ianpatridge/swift-log-**syslog**](https://github.com/ianpartridge/swift-log-syslog) – a [syslog](https://en.wikipedia.org/wiki/Syslog) backend,
```

Comment:
patridge -> partridge

Reply:
Whoops, terribly sorry about this 🙇 Fixing right away.